app/helpers/foreman_ansible/ansible_roles_helper.rb

```diff
@@ -5,5 +5,9 @@ def ansible_roles
       ForemanAnsible::RolesImporter.import!
       AnsibleRole.all
     end
+
+    def available_ansible_roles(target)
```

Comment:
Looks like a good fit for a instance method on Host/Hostgroup `.available_roles` , through a concern they both can share it or something